frontends/verilog/verilog_parser.y

```diff
@@ -643,7 +643,25 @@ wire_type_token_io:
 		astbuf3->is_output = true;
 	};
 
+wire_type_signed:
```

Comment:
Perhaps `wire_type_signedness` to match up with other code?

Reply:
Fixed.